js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -189,6 +189,9 @@ const KeyboardNavigationController = core.ViewController.inherit({
             const columnsResizerController = this.getController('columnsResizer');
             const isColumnResizing = !!columnsResizerController && columnsResizerController.isResizing();
             if(!isCurrentRowsViewClick && !isEditorOverlay && !isColumnResizing) {
+                const targetInsideFocusedView = this._focusedView ? $target.parents().filter(this._focusedView.element()).length > 0 : false;
+
+                !targetInsideFocusedView && this._resetFocusedCell(this._getFocusedCell(), true);
```

Comment:
Maybe "if" will be more readable?
Also _resetFocusedCell will get focused cell by default if first argument is not defined. Maybe . We also have the same logic (get focused cell and reset it) in the _resetFocusedView method. 
I think that first argument in _resetFocusedCell can be removed at all because it is never used.

Reply:
I'm not sure that "if" will be more "readable". As for the first parameter of the _resetFocusedCell, you are right, it can be removed.